packages/js/examples/react-audio-widgets/stories/Audio.stories.js

```diff
@@ -5,20 +5,32 @@ import WebDialerComponent from './components/WebDialer';
 
 export default {
   title: 'Audio',
-  args: {
-    callerName: 'Caller ID Name',
-    callerNumber: 'Caller ID Number',
-    disableMicrophone: false,
-    username: process.env.STORYBOOK_USERNAME,
-    password: process.env.STORYBOOK_PASSWORD,
-    defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
-  },
+ 
 };
 
 export const ClickToCall = (args, storybook) => (
   <ClickToCallComponent {...args} {...storybook.globals.telnyxAuth} />
 );
 
+ClickToCall.args = {
+  token: 'JWT_TOKEN',
+  callerName: 'Caller ID Name',
+  callerNumber: 'Caller ID Number',
+  username: process.env.STORYBOOK_USERNAME,
+  password: process.env.STORYBOOK_PASSWORD,
+  defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
+};
+
 export const WebDialer = (args, storybook) => (
   <WebDialerComponent {...args} {...storybook.globals.telnyxAuth} />
 );
+
+WebDialer.args = {
+  token: 'JWT_TOKEN',
+  callerName: 'Caller ID Name',
+  callerNumber: 'Caller ID Number',
+  disableMicrophone: false,
+  username: process.env.STORYBOOK_USERNAME,
+  password: process.env.STORYBOOK_PASSWORD,
+  defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
```

Comment:
Do we need the default value `'18004377950'`?

Reply:
yes, it is a echo number for test.